storage/src/iroh.rs

```diff
@@ -163,19 +165,28 @@ impl Storage for IrohStorage {
         // There is an issue to track it https://github.com/recallnet/entanglement/issues/27
         let stream = chunked_bytes_stream(bytes.into(), 1024 * 64).map(Ok);
 
-        let progress = self
+        let batch = self
             .client()
             .blobs()
-            .add_stream(stream, SetTagOption::Auto)
+            .batch()
             .await
             .map_err(|e| StorageError::StorageError(storage::wrap_error(e)))?;
 
-        let blob = progress
-            .finish()
+        let temp_tag = batch
+            .add_stream(stream)
             .await
             .map_err(|e| StorageError::StorageError(storage::wrap_error(e)))?;
 
-        Ok(blob.hash.to_string())
+        let hash = *temp_tag.hash();
+        let new_tag = iroh::blobs::Tag(format!("ent-{hash}").into());
```

Comment:
can we include the upload ID here? it would change the `upload_bytes` interface, but i think that's fine... we're the only users of this thing

Reply:
see https://github.com/recallnet/ipc/pull/558